### DISCUSSION TITLE
ABSOLUTE运行问题

### DISCUSSION THREAD

**Discussion by YYBernayu** [ShixiangWang/self-study#97] [Q&A] [answered]
翔哥好！
我目前在使用自测WES数据走流程，已经按照教程整理了segment文件准备跑absolute，但是始终报错如下，使用的文件也上传了，我看有好几个人遇到了这样的问题，似乎是segment的范围不对？都没有找到确切的解答，感谢解惑！
![de3f83b0cb24b9127376a59b71b706d](https://user-images.githubusercontent.com/103264105/162449021-791af874-e1ab-4457-a770-255c49aeabd6.png)
[seg3.txt](https://github.com/ShixiangWang/MessageBoard/files/8452449/seg3.txt)

**Comment by YYBernayu** [comment]
如果解决困难不知道是否还有absolute的示例文件可以参考，genepattern网站上已经没有那个文件了：（

**Comment by ShixiangWang** [comment]
数据本身的问题 参考 https://github.com/ShixiangWang/DoAbsolute/issues/12 看看能不能解决。

**Comment by ShixiangWang** [comment, marked as answer]
https://github.com/ShixiangWang/DoAbsolute 里面保存了示例数据，你可以参考

**Comment by YYBernayu** [comment]
应该还是对公司给的初步结果理解不到位，探针数有问题，感谢解答！
